sys/net/link_layer/csma_sender/csma_sender.c

```diff
@@ -36,13 +36,13 @@
 
 
 /** @brief Current value for mac_min_be parameter */
-static uint8_t mac_min_be = MAC_MIN_BE_DEFAULT;
+static uint8_t mac_min_be = CSMA_SENDER_MIN_BE_DEFAULT;
```

Comment:
Do these (this one and the other global variables below) *have* to change at run-time? Otherwise I would just use the constants and make them configurable at compile-time.